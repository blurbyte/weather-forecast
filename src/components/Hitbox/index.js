import styled from 'styled-components';

const Hitbox = styled.button`
  display: flex;
  justify-content: center;
  align-items: center;
  width: 4rem;
  height: 4rem;
  border: none;
  border-radius: 0;
  background: none;
  outline: none;
  cursor: pointer;
  margin: 0;
  padding: 0;
  -webkit-tap-highlight-color: rgba(255, 255, 255, 0.15);
`;

export default Hitbox;
