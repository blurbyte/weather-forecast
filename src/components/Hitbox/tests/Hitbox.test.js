import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Hitbox from '../index';

test('renders correctly', () => {
  const component = renderer.create(<Hitbox />);
  expect(component.toJSON()).toMatchSnapshot();
});
