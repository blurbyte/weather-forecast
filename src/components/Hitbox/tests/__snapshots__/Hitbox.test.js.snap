// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 4rem;
  height: 4rem;
  border: none;
  border-radius: 0;
  background: none;
  outline: none;
  cursor: pointer;
  margin: 0;
  padding: 0;
  -webkit-tap-highlight-color: rgba(255,255,255,0.15);
}

<button
  className="c0"
/>
`;
